internal/artifacts/module_arch_test.go

```diff
@@ -103,7 +103,7 @@ builders:
 				Ω(err).Should(Succeed())
 				moduleM1, err := mtadObj.GetModuleByName("m1")
 				Ω(err).Should(Succeed())
-				Ω(moduleM1.Path).Should(Equal(getTestPath("result", "data.zip")))
+				Ω(moduleM1.Path).Should(Equal("data.zip"))
```

Comment:
There should be a similar check (for the path in mtad.yaml) for when the target folder is not set